Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/AchievementBucketModel.kt

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package org.dolphinemu.dolphinemu.features.settings.model
+
+import androidx.lifecycle.ViewModel
+import kotlin.collections.ArrayList
+
+class AchievementBucketModel(val numAchievements : Int, val label : String, val subsetId : Int, val bucketType : Int) {
```

Comment:
You should have definitions somewhere for what values can be used for bucketType.

Reply:
Also, you don't need to put the word model in the class name.